src/main/java/com/ericsson/ei/mongo/MongoDBHandler.java

```diff
@@ -123,7 +123,7 @@ public void insertDocument(String dataBaseName, String collectionName, String in
                 }
             }
         } catch (Exception e) {
-            LOGGER.error("Failed to retrieve documents.", e);
+            LOGGER.error("Failed to retrieve documents as {}", e.getMessage());
```

Comment:
This change means we don't see the stacktrace anymore in the logs...  I at least want to log which collection it attempted to find documents in, so we know where to start debugging.